ada/testsuite/ada/nameres.adb

```diff
@@ -1299,9 +1330,27 @@ procedure Nameres is
          end;
       end if;
 
+      Put_Line ("Done.");
+
+      if Args.Memory.Get then
+         Ada.Text_IO.Put_Line
+           ("Memory allocation at end (MB): "
+            & Integer'Image (Integer (Watermark.Current) / 1024 / 1024));
```

Comment:
I suggest to remove the space after the colon,  given that the  number is always positive and thus that `'Image` will always return a string with a leading space.